pipelines/build/devkit/build_devkit.groovy

```diff
@@ -118,8 +118,15 @@ node(params.DEVKIT_BUILD_NODE) {
 
     if (params.DOCKER_IMAGE != "") { 
         // Build within docker container
-        docker.image(params.DOCKER_IMAGE).pull()
-        docker.image(params.DOCKER_IMAGE).inside() {
+        if (!("${params.DOCKER_IMAGE}".contains('rhel'))) {
+            docker.image(params.DOCKER_IMAGE).pull()
+        }
+        String dockerRunArg=""
+        // Add extra mapping for Marist s390x machiens if running podman
```

Comment:
```suggestion
        // Add extra mapping for Marist s390x machines if running podman
```

Reply:
comment is probably a little specific to our infra (don't need to mention Marist?)

Reply:
The UIDs were using are specific to our infra so this was more of an eyecatcher for the future. Marist are the only ones we're doing builds under podman on, but I've changed it to be more generic. This is an interim solution anyway ;-)